server/index.js

```diff
@@ -86,7 +86,9 @@ app.get("/v2/query", async (req, res) => {
     radius: query.radius,
     page: query.page
   }
-  let results = await stores.findNearbyStores(params);
+  let nearbyResults = await stores.findNearbyStores(params);
+  let storeResults = await stores.findNearbyStoreByStoreId(query.storeId)
```

Comment:
I think we should have this in a different API endpoint, instead of overloading this nearby API. The stores we want to return this case are near to the selected store -- and so the lat,lng that we use to filter should be the lat,lng of the selected store. Having this as a separate endpoint feels neater to me.

Reply:
So here's my reasoning, of keeping it part of the nearby api.

When the `storeId` is passed it returns the store and **nearby stores around** it.

Reply:
Yes, and it's also returning `nearbyResults` for a lat/lng which is not relevant. I don't see why we need to do this additional work :)

```js
let nearbyResults = await stores.findNearbyStores(params);
```